app/src/main/index.js

```diff
@@ -197,9 +197,35 @@ function resetTrayIcon() {
   mainWindow.setAlwaysOnTop(false);
 }
 
+const animateIcon = () => new Promise(resolve => {
+  const interval = 40;
+  let i = 0;
+
+  const next = () => {
+    setTimeout(() => {
+      const number = String(i++).padStart(5, '0');
```

Comment:
I considered it, but decided not to since it's not going to be reused anyway and it would be hardcoded to our use-case here. What if we in the future need an animation without `Template` or using a different naming scheme. Probably better to wait until we have a need. And it's only a few characters. Also, in the future, I want rather refactor this into a module that accepts an APNG file and then animates it. That way we wouldn't need all these individual images.